app/angular/src/client/preview/angular/helpers.ts

```diff
@@ -2,46 +2,45 @@ import {
   enableProdMode,
   NgModule,
   Component,
-  NgModuleRef,
-  ApplicationRef,
   CUSTOM_ELEMENTS_SCHEMA
 } from "@angular/core";
+
 import { platformBrowserDynamic } from "@angular/platform-browser-dynamic";
 import { BrowserModule } from "@angular/platform-browser";
 import { AppComponent } from "./components/app.component";
 import { ErrorComponent } from "./components/error.component";
 import { NoPreviewComponent } from "./components/no-preview.component";
 import { STORY } from "./app.token";
+import { getAnnotations, getParameters, getPropMetadata } from './utils';
+
 let platform = null;
 let promises = [];
 
 // Taken from https://davidwalsh.name/javascript-debounce-function
```

Comment:
We already have `lodash.debounce` in the project. Why not using it.

Reply:
If we have it available than let's use it. I was under the impression that we didn't :)